src/main/java/com/example/java/maven/gameBlackJack/Game.java

```diff
@@ -0,0 +1,96 @@
+package com.example.java.maven.gameBlackJack;
+
+public class Game {
+    Player player = new Player(UserInputProvider.getPlayersName());
+    Croupier croupier = new Croupier();
+    private final Deck deck;
+    private final static String INSTRUCTIONS =
+            ('\n' + "INSTRUCTIONS:" + '\n' +
+                    "To take card (hit)    - insert H" + '\n' +
+                    "To do nothing (stand) - insert S"
+            );
+    private final static String INVALID_INPUT = ("Invalid input! Please stick to instructions!");
+    private final String playerLost = ("No luck this time! You lost, " + player.name + "!");
+    private final String playerWon = ("You won, " + player.name + ", you lucky guy!");
+    private final static String CROUPIER_TURN = ("Croupier's turn now.");
+    private final static String CROUPIER_LOST = ("Croupier lost.");
+    private final static String CROUPIER_WON = ("Croupier won.");
+
+
+    public Game() {
+        this.deck = new Deck();
+        initiateGame();
+        while (playersDecision()) {
+            playersDecision();
+        }
+        gameFinishing();
+    }
+
+    public void initiateGame() {
+        player.hand.add(deck.getCard());
```

Comment:
you are making 2 times same thing.... for loop? and for me it will be easier to read this code, when this method would be named: dealCards(), and in separate method we will have `printPlayers()` and `printInstructions()`

Reply:
Done